hr_gamification_email_notification/data/badge.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@Khalid-SerpentCS Can you rename the file with the model of the data it contains `mail_template.xml`?